pkg/sql/catalog/internal/catkv/catalog_reader.go

```diff
@@ -140,6 +143,24 @@ func (cr catalogReader) ScanAll(ctx context.Context, txn *kv.Txn) (nstree.Catalo
 	return mc.Catalog, nil
 }
 
+// ScanAllNamespaceEntries is part of the CatalogReader interface.
+func (cr catalogReader) ScanAllNamespaceEntries(
+	ctx context.Context, txn *kv.Txn,
+) (nstree.Catalog, error) {
+	var mc nstree.MutableCatalog
+	log.Eventf(ctx, "fetching all descriptors and namespace entries")
```

Comment:
s/fetching all descriptors and namespace entries/fetching all namespace entries/